lib/matplotlib/tests/test_ticker.py

```diff
@@ -240,7 +237,7 @@ def _sub_labels(axis, subs=()):
     assert label_test == label_expected
 
 
-@cleanup(style='default')
+@pytest.mark.style('default')
```

Comment:
Indeed, they do… I thought I had seen some classic being converted to default, but I guess we'll see this once the tests run on travis anyways.